src/ImageGallery.react.jsx

```diff
@@ -351,6 +352,7 @@ export default class ImageGallery extends React.Component {
                       onClick={this._wrapClick(swipeNext)}/>
                   ],
                 <Swipeable
+                  className='image-gallery-swipe'
```

Comment:
I will be removing the swipe soon, and adding a better swipe experience, so this would get removed very soon.